src/models/vpn/openvpn/openvpn.service.ts

```diff
@@ -122,9 +122,19 @@ export class OpenVPNService extends Service {
     }
 
     public getCustomizedConfig(): OpenVPNUpdateableConfig {
+        
+        const openvpnConfigFile: string = path.join(this._app.config.get('openvpn').history.data_dir, 'config.json');
```

Comment:
`loadCustomizedConfig` is where the custom config is loaded. This method is used to return only the settings which can be customized.

Reply:
This method is called when the configuration needs to be displayed. In the case that config.json is deleted, 'show' does not return the default configuration.

Reply:
Yes, part of the issue was if config.json was manually deleted, when click configuration again, the default configuration would appear.

Reply:
Ok, in that case you shouldn't reinvent the wheel. Call to `this._config = await this.loadCustomizedConfig(this._config);` in order to "reload" the config. This must be done here but should be called in `archiveHistory` in order to "reload" the config before an archive operation.

Reply:
Notice that `getCustomizedConfig` becomes `async`.

Reply:
> Ok, in that case you shouldn't reinvent the wheel. Call to `this._config = await this.loadCustomizedConfig(this._config);` in order to "reload" the config. This must be done here but should be called in `archiveHistory` in order to "reload" the config before an archive operation.

Sorry, you should use `this._config = this.loadCustomizedConfig(this._app.config.get('openvpn'));`